frontend/src/components/Graph/index.tsx

```diff
@@ -181,6 +181,9 @@ function Graph({
 							},
 						},
 						position: 'custom',
+						itemSort(a, b) {
```

Comment:
can we have a better name here a,b does not highlight much

Reply:
Sure